training/nc_train.xml

```diff
@@ -0,0 +1,55 @@
+  <AddressString><AddressNumber>410</AddressNumber> <StreetName>PIEDMONT</StreetName> <StreetNamePostType>AVE</StreetNamePostType> <OccupancyIdentifier>#</OccupancyIdentifier> <OccupancyType>APT</OccupancyType> <OccupancyIdentifier>2</OccupancyIdentifier> <PlaceName>SHELBY</PlaceName> <ZipCode>28150</ZipCode></AddressString>
+  <AddressString><AddressNumber>5191</AddressNumber> <StreetName>RIVER</StreetName> <StreetNamePostType>RD</StreetNamePostType> <StreetName>LIBERTY</StreetName> <StreetName>GROVE</StreetName> <StreetNamePostType>RD</StreetNamePostType> <PlaceName>NORTH</PlaceName> <PlaceName>WILKESBORO</PlaceName> <ZipCode>28659</ZipCode></AddressString>
+  <AddressString><AddressNumber>61</AddressNumber> <StreetName>WOODROW</StreetName> <StreetName>W</StreetName> <StreetName>WELCH</StreetName> <StreetNamePostType>LN</StreetNamePostType> <PlaceName>CHEROKEE</PlaceName> <ZipCode>28719</ZipCode></AddressString>
+  <AddressString><AddressNumber>2296</AddressNumber> <StreetName>COMPASS</StreetName> <StreetName>POINTE</StreetName> <StreetName>SOUTH</StreetName> <StreetName>WYND</StreetName> <StreetNamePostDirectional>NE</StreetNamePostDirectional> <PlaceName>LELAND</PlaceName> <ZipCode>28451</ZipCode></AddressString>
+  <AddressString><AddressNumber>340</AddressNumber> <StreetName>JAMES</StreetName> <StreetName>W</StreetName> <StreetName>GRAHAM</StreetName> <StreetNamePostType>RD</StreetNamePostType> <PlaceName>MAXTON</PlaceName> <ZipCode>28364</ZipCode></AddressString>
+  <AddressString><AddressNumber>545</AddressNumber> <StreetName>LIBERTY</StreetName> <StreetNamePostType>ST</StreetNamePostType> <OccupancyIdentifier>#</OccupancyIdentifier> <OccupancyType>APT</OccupancyType> <OccupancyIdentifier>25</OccupancyIdentifier> <PlaceName>DURHAM</PlaceName> <ZipCode>27701</ZipCode></AddressString>
+  <AddressString><AddressNumber>337</AddressNumber> <StreetName>HOBSON</StreetName> <StreetNamePostType>DR</StreetNamePostType> <OccupancyIdentifier>#</OccupancyIdentifier> <OccupancyType>APT</OccupancyType> <OccupancyIdentifier>5</OccupancyIdentifier> <PlaceName>MOCKSVILLE</PlaceName> <ZipCode>27028</ZipCode></AddressString>
+  <AddressString><AddressNumber>100</AddressNumber> <StreetName>PARK</StreetName> <StreetName>CIRCLE</StreetName> <StreetName>EAST</StreetName> <StreetNamePostType>RD</StreetNamePostType> <PlaceName>ELIZABETH</PlaceName> <PlaceName>CITY</PlaceName> <ZipCode>27909</ZipCode></AddressString>
+  <AddressString><AddressNumber>322</AddressNumber> <StreetNamePreDirectional>E</StreetNamePreDirectional> <StreetName>SUMTER</StreetName> <StreetNamePostType>ST</StreetNamePostType> <OccupancyIdentifier>#</OccupancyIdentifier> <OccupancyType>APT</OccupancyType> <OccupancyIdentifier>1</OccupancyIdentifier> <PlaceName>SHELBY</PlaceName> <ZipCode>28150</ZipCode></AddressString>
```

Comment:
I agree that this seems like a strange data entry system, but it is common in the North Carolina Secretary of State voter rolls, many addresses have them that way. I do not know if this is common across other SoSs.